package.json

```diff
@@ -194,7 +195,7 @@
     "assertion-error-formatter": "^3.0.0",
     "capital-case": "^1.0.4",
     "cli-table3": "^0.6.0",
```

Comment:
Please upgrade cli-table3 to 0.6.1 too - see https://github.com/cli-table/cli-table3/pull/251

Reply:
Updated!